include/oneapi/dpl/pstl/hetero/dpcpp/execution_sycl_defs.h

```diff
@@ -201,7 +198,7 @@ class fpga_policy : public device_policy<KernelName>
 
 inline device_policy<> dpcpp_default{__internal::__global_instance_tag{}};
 #        if _ONEDPL_FPGA_DEVICE
-inline fpga_policy<> dpcpp_fpga{__internal::__global_instance_tag{}};
+inline fpga_policy<> dpcpp_fpga{};
```

Comment:
I have dropped the workaround from the predefined FPGA policy. The implementation would incorrectly create a default queue instead of an FPGA queue. Fixing it would require `__queue_holder` to search for FPGA devices, likely enabled by a special construction tag, etc. Given that the ONEDPL_FPGA_DEVICE mode is off by default and requires special environment to work, it seems unlikely that somebody faces the same issue with `dpcpp_fpga` as with `dpcpp_default`.